### PR TITLE
Renamed GODefinitionFile to GOOrganController

### PR DESCRIPTION
**Pull request by oleg68**
After moving organ model files to th `model` subdirectory according to the MVC approach
1. Renamed the class `GODefinitionFile` to `GOOrganController` and renamed the the corresponding files
2. Renamed the 'organfile` and `m_organfile` variables to `organController` and `m_OrganController`.

This is just renaming. No Grandrgue behavior should change.

### CODE REVIEW

File: src/grandorgue/sound/GOSoundEngine.h [hunk 1]
```diff
@@ -81,7 +81,7 @@ class GOSoundEngine {
   GOSoundEngine();
   ~GOSoundEngine();
   void Reset();
-  void Setup(GODefinitionFile *organ_file, unsigned release_count = 1);
+  void Setup(GOOrganController *organ_file, unsigned release_count = 1);
```
Comment (larspalo) [inline src/grandorgue/sound/GOSoundEngine.h:84]: In most other places you've changed to organ_controller, why not here?

File: src/grandorgue/sound/GOSoundEngine.cpp [hunk 2]
```diff
@@ -206,7 +206,7 @@ void GOSoundEngine::ClearSetup() {
 }
 
 void GOSoundEngine::Setup(
-  GODefinitionFile *organ_file, unsigned release_count) {
+  GOOrganController *organ_file, unsigned release_count) {
```
Comment (larspalo) [inline src/grandorgue/sound/GOSoundEngine.cpp:209]: In similar places you've changed the naming to something like organ_controller, why not here?

Comment (oleg68) [reply src/grandorgue/sound/GOSoundEngine.cpp]: Renamed to `organControlled`

File: src/grandorgue/sound/GOSoundEngine.h (hunk 1 above)
Comment (oleg68) [reply src/grandorgue/sound/GOSoundEngine.h]: Renamed to organControlled

**Review by larspalo — APPROVED** [review]

**Comment by oleg68** [comment]
@rousseldenis could you approve this pr?

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**